app/mdx-page/page.js

```diff
@@ -1,12 +1,23 @@
 "use client";
 import Welcome from "@/markdown/welcome.mdx";
+import Welcome2 from "@/markdown/welcome2.mdx";
+import style from "./page.module.css";
+import { useMDXComponents } from "@mdx-js/react";
 
 export default function Page() {
+  const pages = [Welcome, Welcome2];
```

Comment:
read pages maybe via fs?

Reply:
The effort is far too high. I would start with a static approach first.